### ISSUE TITLE
"mloginfo --rsstate" does not show actual host in output

### ISSUE THREAD

**Issue by Steve-Hand** [rueckstiess/mtools#408]
"mloginfo --rsstate" does not how actual host in output.. Instead of a hostname in the "host column" the word " now is shown. 

This problem exists with version 1.2.1 and earlier.

**Comment by stennie** [comment]
Hostnames display as expected with 2.6.12:

```
Jul 17 16:06:21    Webscale.local:27017 (self)    replSet info electSelf 0
Jul 17 16:06:21    Webscale.local:27017 (self)    PRIMARY
Jul 17 16:06:33    Webscale.local:27018           SECONDARY
Jul 17 16:06:33    Webscale.local:27019           SECONDARY
```

No host info displayed for 3.0.12 or 3.2.7:

```
Jul 17 16:06:45    now     SECONDARY
Jul 17 16:06:45    now     SECONDARY
Jul 17 16:06:47    now     PRIMARY
```

**Comment by Steve-Hand** [comment]
Fixed with the merging #427
